### PR TITLE
docs(identity): update Keycloak assets and steps to v22

### PR DESCRIPTION
**Pull request by Ben-Sheppard**
## Description

Closes https://github.com/camunda-cloud/identity/issues/2045

The release of Identity 8.3 will mean that our supported Keycloak versions are v21 and v22, this means that to provide a good (and accurate) guide experience all the Keycloak assets and links should be for the correct version.

I can see that @christinaausley has already handled a lot of the links (thank you!) so this PR is mainly focused on the images and some changes in steps based on those images.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md [hunk 1]
```diff
@@ -17,32 +17,34 @@ In this guide, we'll demonstrate how to connect Identity to your existing Keyclo
 To connect Identity to an existing Keycloak instance, take the following steps:
 
 1. Log in to your Keycloak Admin Console.
-2. Select the realm you would like to connect Identity to. In our example, this is **Test Realm**.
+2. Select the realm you would like to connect Identity to. In our example, this is **camunda-platform**.
```
Comment (christinaausley) [inline docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md:20]: FYI we are in the midst of renaming -- are we wanting to rename this to **camunda** @dlevy-camunda, or keep as-is?

**Review by christinaausley — COMMENTED** [review]
Nice WORK on this! One comment to resolve regarding renaming 😅

File: docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md (hunk 1 above)
Comment (akeller) [reply docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md:20]: @christinaausley can you raise this in the [tmp-camunda-product-name](https://camunda.slack.com/archives/C05P016GR5M) channel? I'd like to see input from Mo and Max, not necessarily for this example but other programmatic examples. Basically how granular do we want to go with renaming?

**Comment by akeller** [comment]
Looks like this is part of the alpha. Is this ready to merge? Or is it ok to wait a bit, so long as it's merged for the 8.3 release?

**Comment by Ben-Sheppard** [comment]
> Looks like this is part of the alpha. Is this ready to merge? Or is it ok to wait a bit, so long as it's merged for the 8.3 release?

@akeller - It did make it into the alpha, but its more visible release will be with 8.3, the reason I haven't merged this yet as there is one question around the naming I've chosen [here](https://github.com/camunda/camunda-platform-docs/pull/2556#discussion_r1319981594) that I believe is yet to be answered.

Personally I don't really like leaving PRs open for a bit so I'd be happy to merge this and update if necessary later but I don't think thats completely my call

File: docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md (hunk 1 above)
Comment (akeller) [reply docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md:20]: @christinaausley can you convert this to an issue and follow up? This shouldn't block this PR.

**Comment by akeller** [comment]
> the reason I haven't merged this yet as there is one question around the naming I've chosen [here](https://github.com/camunda/camunda-platform-docs/pull/2556#discussion_r1319981594) that I believe is yet to be answered.

We can always follow up on this with another PR. Let's merge this if it's ready @Ben-Sheppard 😎

**Pull request merged**